bndtools.core/src/bndtools/editor/project/RunRequirementsPart.java

```diff
@@ -474,9 +512,7 @@ protected boolean performSelectionDrop(ISelection data, Object target, int locat
             }
 
             if (!adding.isEmpty()) {
```

Comment:
Actually, I think the best solution is to simply replace the List with a Set. We don't need a dialog popup.